### ISSUE TITLE
rounded corner unpolished

### ISSUE THREAD

**Issue by OliverScholtz** [ChrisAnd1998/TaskbarXI#46]
For example, light theme with darker wallpaper you see rounded corner unpolished. The corners have to been smoothed for better look.

**Comment by yxles-dev** [comment]
yeah it looks pixelated compared to rounded corners that Win11 use throughout the os

**Comment by ChrisAnd1998** [comment]
This is a limitation in Windows. But... i may have found a way to do it better.
